client/rest-high-level/src/test/java/org/elasticsearch/client/indexlifecycle/SetIndexLifecyclePolicyRequestTests.java

```diff
@@ -69,6 +63,16 @@ protected SetIndexLifecyclePolicyRequest mutateInstance(SetIndexLifecyclePolicyR
         return newRequest;
     }
 
+    private SetIndexLifecyclePolicyRequest copyInstance(final SetIndexLifecyclePolicyRequest original) {
+        SetIndexLifecyclePolicyRequest copy = new SetIndexLifecyclePolicyRequest(original.policy(), original.indices());
+        copy.indicesOptions(original.indicesOptions());
```

Comment:
Doesn't this need to copy the timeouts also?

Reply:
It probably should to be strictly correct, but this made me check (since obviously this doesn't make the `equals()` test fail) and as far as I can tell, no (or very, very few) classes that inherit the timeout parameters from a superclass check them in the `.equals()` method. Is that something we should be concerned about?

(cc @hub-cap)